app/views/branches/_form_expressions.html.erb

```diff
@@ -1,5 +1,7 @@
 <%= conditional.fields_for :expressions do |expression| %>
-  <%= expression.select :component,
-    @branch.previous_questions,
-    include_blank: '--Select a question--' %>
+  <div data-index-expression="<%= expression.index %>">
```

Comment:
```suggestion
  <div data-expression-index="<%= expression.index %>">
```

Reply:
Sorry. I only saw the approval 🙈 . Let's change this in the future as this is only to set our tasks running.